crypto/signing/disabled/singlesig/disabled.go

```diff
@@ -0,0 +1,24 @@
+package singlesig
+
+import "github.com/ElrondNetwork/elrond-go/crypto"
+
+const signature = "signature"
+
+// Disabled represents a disabled singlesigner implementation
+type Disabled struct {
```

Comment:
Disabled what ?

Reply:
singlesig.Disabled, I think it's pretty obvious

Reply:
DisabledSingleSig ? to keep the same pattern used above in crypto/signing/disabled/point.go -> DisabledPoint
and I think you can rename this file to singlesig.go and move it in crypto/signing/disabled/ folder alongside point.go ?

Reply:
done, renamed. I would keep them as they are as to comply with the rest of the crypto package implementations.